Curs04/src/views/index.js

```diff
@@ -1,8 +1,96 @@
 
 window.onload = function(){
-    const result = [];
-    result.push(LoginPage({}))
     const app = document.querySelector('#app');
+    // call LoginPage function and display LoginPage using innerHTML method
     app.innerHTML = LoginPage();
+
+    // get the form by his ID
+   const loginFormSection = document.getElementById("form-login")
+
+    const myObj = {}
+
+   const goToNewEvaluationPage = function(event) {
+        event.preventDefault();
+        // call NewEvaluationPage() which will render the New-evaluation page
+        app.innerHTML = NewEvaluationPage()
+
+        // get evaluationButton by his ID
+        const evaluationButton = document.getElementById("evaluation-page")
+
+        // add "click" event listener
+        evaluationButton.addEventListener("click", goToEvaluationsPage)
+
+        const getSelectorAll = document.getElementsByTagName("select")
+        getSelectorAll.forEach(function(el) {
+           const nameSelector = el.name
+           const valueSelector = el.value  
+
+           myObj[nameSelector] = valueSelector
+
+           console.log(myObj)
+        })
+    }
+
+    // go to evaluation page function
+    const goToEvaluationsPage = function() {
+        // call EvaluationPage function and display EvaluationPage using innerHTML method
+        app.innerHTML = EvaluationsPage();
+        const newEvaluationButton = document.getElementById("new-evaluation-page")
+
+        // add click event listener on newEvaluationButton
+        newEvaluationButton.addEventListener("click", goToNewEvaluationPage)
```

Comment:
like how the names are representative to their contents/purpose

Reply:
newEvaluationButton is the variable which contains our ID element (the ID of the button)
After that I added a click event listener and I called the goToNewEvaluationpage function where the new-evaluation page will be rendered.

Reply:
yes, it is intuitive and simple to understand.

Reply:
At the moment yes. 👍